frontend/src/stores/QuestionStore.ts

```diff
@@ -68,11 +68,11 @@ export const useQuestionStore = defineStore('question', () => {
 
   function updateLabelsAtConclusion() {
     /**
-     * This function will change all the "Nader te bepalen" labels to "Niet van Toepassing" when
+     * This function will change all the "nader te bepalen" labels to "niet van toepassing" when
      * the conclusion of the decision tree has been reached.
      */
     for (let key in LabelsByCategory.value) {
-      if (JSON.stringify(LabelsByCategory.value[key]) === JSON.stringify(['Nader te bepalen'])) {
+      if (JSON.stringify(LabelsByCategory.value[key]) === JSON.stringify(['nader te bepalen'])) {
         LabelsByCategory.value[key] = ['Niet van toepassing']
```

Comment:
Can you also do the Niet van toepassing?

Reply:
🤦  of course, thanks